lib/logstash/outputs/lumberjack.rb

```diff
@@ -73,6 +74,10 @@ def flush(events, close = false)
       retry
     end # begin
   end
+
+  def close
+    buffer_flush(:force => true) 
```

Comment:
@jsvd I will add back the `logstash::shutdown` check but I prefer to use #close to call the buffer_flush, this is the same behavior as the `logstash-output-elasticsearch` and I think we should keep the same logic.